apps/redi-talent-pool/src/pages/app/browse/BrowseJobseeker.tsx

```diff
@@ -396,13 +397,14 @@ export function BrowseJobseeker() {
             if (!isFavorite && onlyFavorites) return
 
             return (
-              <Columns.Column mobile={{ size: 12 }} tablet={{ size: 6 }}>
+              <Columns.Column size={12}>
                 <JobListingCard
                   key={jobListing.id}
-                  jobListing={jobListing as unknown as any}
+                  jobListing={jobListing}
                   toggleFavorite={handleFavoriteJobListing}
                   isFavorite={isFavorite}
                   linkTo={`/app/job-listing/${jobListing.id}`}
+                  timeFooter={moment(jobListing.createdAt).fromNow()}
```

Comment:
As we already pass the `jobListing` item to JobListingCard as a prop, we already have the information `createdAt` there. So probably we don't need this extra prop `timeFooter` and we can directly do the moment trick inside the JobListingCard. What do you think?